<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tarea armas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tituloInventario">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inventario de armas</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetArmas">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>711</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAgregar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEliminar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="tituloInventario_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Filtrar por:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreBuscar">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>70</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetTipoBuscar">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>110</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modificador</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetModBuscar">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>210</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAgregar_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>310</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEliminar_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>310</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar todas</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuInventario"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
